.github/workflows/integ-tests.yml

```diff
@@ -111,7 +111,7 @@ jobs:
     with:
       script-path: examples/launch_transfer_learning_example.py
 
-  launch_height_simple:
```

Comment:
Note that ``launch_height_simple.py`` is not an executable Python script, so does not make sense to have it here. It is also duplicated by others, like ``launch_height_baselines``.

Reply:
I just ran `python examples/launch_height_simple.py` and got the following output: 

```
% python examples/launch_height_simple.py
--------------------
Resource summary (last result is reported):
 trial_id     status  iter  width  height  epochs  epoch  mean_loss  worker-time
        0    Stopped    41     10      10     100   41.0   1.243902     4.237480
        1    Stopped    41     13      11     100   41.0   1.288679     4.234309
        2  Completed   100     10       1     100  100.0   0.200000    10.316471
        3    Stopped    41      2      10     100   41.0   2.111111     4.232994
        4    Stopped    41     13      14     100   41.0   1.588679     4.220124
        5    Stopped    41     12      16     100   41.0   1.804082     4.220239
        6    Stopped    41      1       3     100   41.0   2.300000     4.216881
        7    Stopped    41     17       5     100   41.0   0.644928     4.233574
        8    Stopped    41     14       8     100   41.0   0.975439     4.229480
        9    Stopped    41      6       7     100   41.0   1.100000     4.222577
       10    Stopped    41     19      13     100   41.0   1.429870     4.238196
       11    Stopped    90     16       2     100   90.0   0.269735     9.291159
       12    Stopped    41     17       7     100   41.0   0.844928     4.223391
       13    Stopped    41     18       5     100   41.0   0.636986     4.221838
       14    Stopped    41      1      11     100   41.0   3.100000     4.227698
       15    Stopped    41     20      19     100   41.0   2.023457     4.226891
       16    Stopped    41     18      17     100   41.0   1.836986     4.225258
       17 InProgress     0     20       2     100      -          -            -
       18 InProgress     0     20      17     100      -          -            -
       19 InProgress     0     15      16     100      -          -            -
       20 InProgress     0     17      16     100      -          -            -
4 trials running, 17 finished (1 until the end), 30.31s wallclock-time

mean_loss: best 0.2 for trial-id 2

```

So I believe it is executable. 

I'm fine with removing it if it's redundant / covered by other tests, just to my knowledge, it is indeed executable.

Reply:
It is just in examples because it is included in our docs. If it runs, I'll keep it in there.